packages/admin/components/composed/ordering/EntityOrderingAddForm/EntityOrderingAddForm.tsx

```diff
@@ -1,54 +1,53 @@
 import * as React from "react";
 import { graphql, useFragment } from "react-relay";
-import { useTranslation } from "react-i18next";
 import MutationForm, {
   useRenderForm,
   useToVariables,
   Forms,
 } from "components/api/MutationForm";
+import { convertToSlug } from "helpers";
 
 import type {
   EntityOrderingAddFormMutation,
   CreateOrderingInput,
   OrderDefinitionInput,
   OrderingSelectDefinitionInput,
-  OrderingFilterDefinitionInput,
+  OrderingDirectSelection,
 } from "@/relay/EntityOrderingAddFormMutation.graphql";
 import type { EntityOrderingAddFormFragment$key } from "@/relay/EntityOrderingAddFormFragment.graphql";
 
 const description = `Select descendants to be included in the ordering. Use Direct
 Descendants to select only immediate children, for example journal
-issues but not their articles. Descendants must be included for at
-least one type of entity.`;
+issues but not their articles.`;
 
 export default function EntityOrderingAddForm({
   data,
   onSuccess,
   onCancel,
 }: Props) {
-  const { t } = useTranslation();
-
   const formData = useFragment<EntityOrderingAddFormFragment$key>(
     fragment,
     data
   );
-  const schemaOptions = formData.schemaVersions;
+  // const schemaOptions = formData.schemaVersions;
   const entity = formData.item ?? formData.collection;
 
-  const getFilter = (
-    items: OrderingSelectDefinitionInput["direct"] = "CHILDREN",
-    collections: OrderingSelectDefinitionInput["direct"] = "CHILDREN"
-  ): OrderingFilterDefinitionInput["schemas"] => {
-    if (items !== "NONE" && collections !== "NONE") return null;
-    if (items === "NONE")
-      return schemaOptions.edges
-        .filter((node) => node.node.kind === "ITEM")
-        .map((node) => node.node.identifier);
-    if (collections === "NONE")
-      return schemaOptions.edges
-        .filter((node) => node.node.kind === "COLLECTION")
-        .map((node) => node.node.identifier);
-  };
+  /*
+   *  const getFilter = (
+   *    items: OrderingSelectDefinitionInput["direct"] = "CHILDREN",
+   *    collections: OrderingSelectDefinitionInput["direct"] = "CHILDREN"
+   *  ): OrderingFilterDefinitionInput["schemas"] => {
+   *    if (items !== "NONE" && collections !== "NONE") return null;
+   *    if (items === "NONE")
+   *      return schemaOptions.edges
+   *        .filter((node) => node.node.kind === "ITEM")
+   *        .map((node) => node.node.identifier);
+   *    if (collections === "NONE")
+   *      return schemaOptions.edges
+   *        .filter((node) => node.node.kind === "COLLECTION")
+   *        .map((node) => node.node.identifier);
+   *  };
+   */
```

Comment:
@jen-castiron Should I remove everything related to the schema filter before merging?

Reply:
Yeah, probably... We don't know if the API will change before we get to it